app_dart/lib/src/datastore/cocoon_config.dart

```diff
@@ -126,6 +126,14 @@ class Config {
     return ServiceAccountCredentials.fromJson(json.decode(rawValue));
   }
 
+  /// The names of autoroller accounts for the repositories.
+  ///
+  /// These accounts should not need reviews before merging.
+  Set<String> get rollerAccounts => <String>{
+    'skia-flutter-autoroll',
+    'engine-flutter-autoroll',
+  };
+
```

Comment:
Not sure if this should go into datastore - these rarely change and it seems like people would rather have values in cocoon.

Trivial to put it in datastore though, LMK.